code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -189,7 +189,7 @@
 	name = "\improper Value-PAK cartridge"
 	desc = "Now with 350% more value!" //Give the Captain...EVERYTHING! (Except Mime and Clown)
 	icon_state = "cart-c"
-	access = ~(CART_CLOWN | CART_MIME)
+	access = ~(CART_CLOWN | CART_MIME | CART_REMOTE_DOOR)

```

Comment:
Why'd you remove that?

Reply:
Technically it removes, since this is a list of carts **NOT** to use